lg_sv/scripts/launcher.py

```diff
@@ -89,7 +89,10 @@ def main():
     slug = (server_type + "__" + "fov-" + str(field_of_view) + "__" + "yaw-" +
             str(yaw_offset) + "__" + "pitch-" + str(pitch_offset) +
             "__" + str(slug_suffix))
-    managed_browser = ManagedAdhocBrowser(url=url, geometry=geometry, slug=slug, kiosk=kiosk)
+
+    # add modify_cors_headers chrome extension to handle the cors error
+    list_of_extensions = ['/opt/ros/melodic/share/lg_sv/extensions/modify_cors_headers']
```

Comment:
This hardcoded path will be misdirected when in a development environment or broken when ROS distribution changes.  Here is a technique for getting the share path for a package:

```
import rospkg

extensions_dir = rospkg.RosPack().get_path('lg_sv') + '/extensions'
```

Reply:
That's right, just fixed those changes @mvollrath